dawn/src/dawn/CodeGen/CodeGen.cpp

```diff
@@ -346,8 +346,7 @@ void CodeGen::addMplIfdefs(std::vector<std::string>& ppDefines, int mplContainer
 
   ppDefines.push_back(makeIfNotDefined("BOOST_RESULT_OF_USE_TR1", 1));
   ppDefines.push_back(makeIfNotDefined("BOOST_NO_CXX11_DECLTYPE", 1));
-  ppDefines.push_back(
-      makeIfNotDefined("GRIDTOOLS_CLANG_HALO_EXTEND", codeGenOptions.MaxHaloPoints));
+  ppDefines.push_back(makeIfNotDefined("GRIDTOOLS_DAWN_HALO_EXTEND", codeGenOptions.MaxHaloPoints));
```

Comment:
shouldn't it be `DAWN_GRIDTOOLS`? And can you rename it to `EXTENT` with a `T`?

Reply:
The `GRIDTOOLS_DAWN` prefix is to mirror the namespace `gridtools::dawn`. Having `dawn::gridtools` would have made things much more complicated, as to access objects within `::dawn` we would have needed to prefix them with `::dawn::`.
Will fix the D instead of T typo.